testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -130,27 +133,22 @@
 /**
  * Contains unit tests for all operators. Each of the methods is named after an
  * operator.
- *
```

Comment:
This file has probably been reformatted by the IDE.

Reply:
Fixed.